applications/SwimmingDEMApplication/python_scripts/swimming_DEM_solver.py

```diff
@@ -106,14 +106,14 @@ def GetHistoryForceQuadratureCounter(self):
             1,
             self.pp.CFD_DEM["basset_force_type"].GetInt())
 
-    def AdvanceInTime(self, step, time):
-        self.step, self.time = self.dem_solver.AdvanceInTime(step, time)
+    def AdvanceInTime(self, time):
```

Comment:
@farrufat-cimne , every time we change the name of a method or the arguments, we should leave the old method with a warning. I think it's the way to go if we don't want an angry Kratos DEM community.

Reply:
the warning is the typical one for deprecated methods.

Reply:
Python do not support overloading functions as C++ do, at least not in an easy way. We could leave some of the arguments with default values but it is not possible in some cases.
As discussed, for now we will leave the old methods and create a new method names.